lib/Partitioner/Partitioner.cpp

```diff
@@ -425,6 +426,38 @@ Expected<DAGListTy> Partitioner::createDAGWithoutPartition(
   }
 
   NodeToFunctionMap mapping;
+  for (auto func : module_->getFunctions()) {
+    mapping.createPartition(func, backendName);
+
+    GraphMemInfo graphMem;
+    graphMem.constMemSize = func->getParent()->getConstantsSize();
+
+    // walk thru all the nodes to compute input and output mem size
```

Comment:
nit: add a full stop (.) after each comment :)

Reply:
Sure thing. I read the guidelines earlier but forgot. Fixed now. Thanks.